data/common.yaml

```diff
@@ -1 +1,8 @@
---- {}
+---
+
+profile_xfs_quota::enable: false
```

Comment:
Do we want this to default to `false`?

My thinking is that if we're including this profile we would expect it to be enabled.

Reply:
So I might be going about it the wrong way, but I switched the default to `false` because we use a role module (e.g.  slurm_compute.pp) to include the profile, and then specify the nodes that have quota enabled xfs mounts through node specific modules.

But maybe its better to have it default true, and in our case add a blanket default false and do what we planned on doing? I don't have much experience with how other control repos are setup so it's hard for me to envision what the most generalized form should look like.

Reply:
I'll defer to you or Jake's perspective on this.

My thinking on this currently...
- This profile is slightly more broad than just for Slurm compute nodes (though initially that would be the role where we plan to use this).
- Generally if I'm including a profile, I would expect its core functionality to be enabled by default.
- If a project wants to include this in a specific role but not use it by default, they should just set `profile_xfs_quota::enable: false` by default for the role, then enable it somehow for the specific nodes that need it.
- I presume a project could even adjust their custom role in their control-repo to only include this profile for hosts that match some condition (e.g. based on a fact, certain matches for hostnames, etc).

Reply:
I'm OK either way as well, but I see Bill's point. Default to true is totally fine.

Reply:
Sounds good, I've reverted the change so it now defaults to true.

Reply:
My thinking with this is not to be nit picky, but just that now is the ideal time to establish defaults rather than them have them flip-flop on users in later version.